src/sparseml/pytorch/torchvision/train.py

```diff
@@ -436,6 +436,9 @@ def collate_fn(batch):
     manager = ScheduledModifierManager.from_yaml(args.recipe_path)
     optimizer = manager.modify(model, optimizer, len(data_loader))
 
+    if manager.learning_rate_modifiers:
+        lr_scheduler = None
```

Comment:
We definitely need (manual) testing for this PR.

Reply:
do we want people going through the cli for lr scheduler? the recipe needs to know if lr is getting changed right?

Reply:
no, we don't want people running through - preferred method is to use the recipe. my point was that line 416 expects there to be something in the state dict to load for the scheduler, just need to account for the fact that in our saved models it may not be there :)